File: Benchmarks/enterprise/java/spring/testcode/BenchmarkTest32869.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest32869 {

    private static String normalize(String v) { return v.strip(); }

    @GetMapping("/BenchmarkTest32869")
    public void BenchmarkTest32869(@RequestHeader("Authorization") String authorization, HttpServletRequest request, HttpServletResponse response) throws Exception {
        String authHeader = authorization != null ? authorization : "";
        String data = normalize(authHeader);
        response.sendError(500, data);
    }
}

File: Benchmarks/enterprise/python/flask/testcode/benchmark_test_32869.py
# SPDX-License-Identifier: Apache-2.0
import os
from flask import request, jsonify


def BenchmarkTest32869():
    graphql_var = (request.get_json(silent=True) or {}).get('variables', {}).get('input', '')
    def normalize(value):
        return value.strip()
    data = normalize(graphql_var)
    os.remove(str(data))
    return jsonify({"result": "success"})
